CompetitionTemplate/tests/competition/subsystems/low_bar_to_low_goal/LowBarLowScoreCommandGroupTest.java

```diff
@@ -0,0 +1,109 @@
+package competition.subsystems.low_bar_to_low_goal;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import competition.subsystems.autonomous.DriveForDistanceCommand;
+import competition.subsystems.autonomous.LowBarScoreCommandGroup;
+import competition.subsystems.autonomous.TurnToHeadingCommand;
+import competition.subsystems.drive.DriveTestBase;
+import competition.subsystems.drive.PoseSubsystem;
+import edu.wpi.first.wpilibj.MockEncoder;
+
+public class LowBarLowScoreCommandGroupTest extends DriveTestBase{
+    LowBarScoreCommandGroup lowScoreCommands;
+    
+    @Before
+    public void setup() {
+        super.setup();
+        this.lowScoreCommands = this.injector.getInstance(LowBarScoreCommandGroup.class);
+    }
+    
+    @Test
+    public void testDriveToTurningPoint() {
+        DriveForDistanceCommand driveForDistanceCommand = this.injector.getInstance(DriveForDistanceCommand.class);
```

Comment:
Unless this is doing something different now, should remove the tests from here that are covered in the other tests files.